redis_fdw.c

```diff
@@ -82,7 +82,6 @@ struct RedisFdwOption
  */
 static struct RedisFdwOption valid_options[] =
 {
-
```

Comment:
Please remove this unrelated whitespace change.

Reply:
No problems, please verify, @dpage !